nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/propertytable/jquery.propertytable.js

```diff
@@ -2001,6 +2021,43 @@
         propertyData.setItems([]);
     };
 
+    /**
+     * Gets all the referenced parameters from the {parameterContext} based on the value of {parameterReference} with matching {sensitive} property
+     *
+     * @param {string} parameterReference
+     * @param {ParameterContextEntity} parameterContext
+     * @param {boolean} sensitive
+     * @returns {ParameterEntity[]}
+     */
+    var getExistingParametersReferenced = function (parameterReference, parameterContext, sensitive) {
+        var parameters = _.get(parameterContext.component, 'parameters', []);
+        var existingParametersReferenced = [];
+
+        if (!_.isNil(parameterReference) && !_.isEmpty(parameters)) {
+            // can't use from common/constants because we are modifying the lastIndex below
+            var paramRefsRegex = /#{(')?([a-zA-Z0-9-_. ]+)\1}/gm;
```

Comment:
Is it necessary to define a capturing group for the `'` character? It seems like the pattern could be adjusted to use a single capturing group:
```suggestion
            var paramRefsRegex = /#{'?([a-zA-Z0-9-_. ]+)'?}/gm;
```

Reply:
@exceptionfactory It's not necessary to define a separate capturing group for `'`. I updated this block of code.

Reply:
@mcgilman Good point. I changed the pattern for these cases.

Reply:
Thanks for noting that background @mcgilman, that makes sense, and thanks for making the changes @mtien-apache.